src/components/routers/SwitchRouter.tsx

```diff
@@ -183,18 +180,15 @@ export const resolveExits = (newCases: CaseElementProps[], previous: Node): Comb
     return { cases, exits, defaultExit: defaultUUID };
 };
 
-export const composeExitMap = (exits: Exit[]): { [uuid: string]: Exit } =>
-    exits.reduce(
-        (map, exit) => {
-            map[exit.uuid] = exit;
-            return map;
-        },
-        {} as { [uuid: string]: Exit }
-    );
-
-export const getListStyle = (isDraggingOver: boolean): { cursor: DragCursor } => ({
-    cursor: isDraggingOver ? DragCursor.move : DragCursor.pointer
-});
+export const getListStyle = (isDraggingOver: boolean, single: boolean): { cursor: DragCursor } => {
```

Comment:
bit hacky, but this ensures the cursor doesn't change when hovering over an un-draggable case. i can handle this more appropriately when I look take care of #130